CHANGELOG.md

```diff
@@ -1,13 +1,37 @@
-# Changelog and release notes
+# Changelog
 
-### 0.2.3 [BREAKING CHANGE]
+_This changelog follows the [keep a changelog][keep-a-changelog]_ format to maintain a human readable changelog.
+
+### [0.3.1][v0.3.1] - 2020-07-29
+
+#### Added
+
+- table of content added to readme
+
+#### Changed
+
+- moved from Mocha to Jest
+- added Prettier for code formatting
+- added Eslint for linting
+- updated CI configuration
+- removed some unused dev dependencies
+- updated dependencies to latest version
+
+#### Fixed
+
+- circular dependency fixed
+- dev dependencies removed from package.json before publishing (no more security warnings)
+- transformer oder is deterministic now (#231)
+- fix prototype pollution issue (#367)
+
```

Comment:
minor doc issues: https://github.com/typestack/class-transformer/pull/382/commits/915159b39fc63342d41853074f7af4b5bdd1f399

Reply:
Thanks! Fixed in 9bd7408.